lib/bolt/project.rb

```diff
@@ -44,13 +45,23 @@ def self.find_boltdir(dir)
 
     def self.create_project(path, type = 'option')
       fullpath = Pathname.new(path).expand_path
+
+      if ENV['BOLT_PROJECT'].nil? && fullpath.world_writable?
```

Comment:
It doesn't seem quite right that we check for the env var here but we _use_ it elsewhere. Maybe we should pass `type='environment'` from the CLI and check _that_ here.

Reply:
That makes wayyy more sense. 👍